src/components/Socials/styles.scss

```
@@ -1,3 +1,64 @@
+// Mobile portrait
 .wrapper {
+	display: flex;
+	justify-content: center;
+	width: 100%;
 	font-size: 1em; //nothing comment to make sure class appears
+	
+		li {
+			width: 1em;
+			height: 1em;
+			list-style: none;
+		}
+
+			a {
```

```suggestion
			.link {
```
We should give this a name in the `renderUtil`